src/popup/components/ReverseAlias.vue

```diff
@@ -0,0 +1,136 @@
+        <br /><br />
+        <label>
+          Receiver:
+          <font-awesome-icon
+            v-b-tooltip.hover.top="'Where do you want to send the email?'"
+            icon="question-circle"
+          />
+        </label>
+        <b-input
+          v-model="receiverEmail"
+          placeholder="First Last &lt;email@example.com&gt;"
+          :disabled="loading"
+        />
+      </div>
+
+      <!-- Created screen -->
+      <div class="m-2 p-2" v-else>
+        <p class="font-weight-bold">Reverse-alias is created:</p>
+        <p>
+          <a
+            v-clipboard="() => createdReverseAlias.reverse_alias_address"
+            v-clipboard:success="clipboardSuccessHandler"
+            v-clipboard:error="clipboardErrorHandler"
+            v-b-tooltip.hover
+            title="Click to Copy"
+            class="cursor"
+          >
+            <span class="text-success">
+              {{ createdReverseAlias.reverse_alias_address }}
+            </span>
+          </a>
+        </p>
+        <small>
+          You can send email from one of these mailbox(es):
+          <ul style="margin-bottom: 0;">
+            <li v-for="mailbox in alias.mailboxes" v-bind:key="mailbox.id">
+              {{ mailbox.email }}
+            </li>
+          </ul>
+          The email will be forwarded to
+          <b>{{ createdReverseAlias.contact }}</b
+          >.<br />
+          The receiver will see <b>{{ alias.email }}</b> as your email
+          address.<br />
+        </small>
+      </div>
+
+      <div class="m-2 p-2">
+        <button
+          class="btn btn-sm btn-primary"
+          @click="createReverseAlias"
+          :disabled="loading || !receiverEmail"
+          v-if="!createdReverseAlias"
+        >
+          Create a reverse-alias
+        </button>
+
+        <button class="btn btn-sm btn-primary" @click="backToMainPage" v-else>
+          <font-awesome-icon icon="arrow-left" />
+          Back
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import SLStorage from "../SLStorage";
+import Navigation from "../Navigation";
+import Utils from "../Utils";
+import { callAPI, API_ROUTE, API_ON_ERR } from "../APIService";
+
+export default {
+  data() {
+    return {
+      alias: SLStorage.getTemporary("alias"),
+      createdReverseAlias: null,
+      loading: false,
+      receiverEmail: "",
+    };
+  },
+  methods: {
+    // Clipboard
+    clipboardSuccessHandler({ value, event }) {
+      Utils.showSuccess(value + " copied to clipboard");
+    },
+
+    clipboardErrorHandler({ value, event }) {
+      console.error("error", value);
+    },
+
+    // Create reverse-alias
+    async createReverseAlias() {
+      const { data } = await callAPI(
+        API_ROUTE.CREATE_REVERSE_ALIAS,
+        {
+          alias_id: this.alias.id,
+        },
+        {
+          contact: this.receiverEmail,
+        },
+        API_ON_ERR.TOAST
+      );
+      this.createdReverseAlias = data;
```

Comment:
there's a field `existed` in `data` that indicates whether this is a the reverse-alias is new or it has been created before and we can then change the wording to reflect that: if the reverse-alias was created before, we can say "you have created this reverse alias before" instead of "reverse-alias is created".

Reply:
Added on https://github.com/simple-login/browser-extension/pull/104/commits/2d67d301fb32cf1a7e9318e98f8afe108c12339b